content/en/user-guide/aws/stepfunctions/index.md

```diff
@@ -0,0 +1,127 @@
+---
+title: "Step Functions"
+linkTitle: "Step Functions"
+categories: ["LocalStack Community"]
+description: Get started with Amazon Web Services Step Functions
```

Comment:
```suggestion
description: Get started with AWS Step Functions
```

Reply:
Even AWS almost never uses anything but the abbreviation so that might just be confusing to people.

Would also remove these from most of the text.